### DISCUSSION TITLE
Twilio integration

### DISCUSSION THREAD

**Discussion by Rishabhh17** [chatwoot/chatwoot#5787] [Self hosted]
Hi @jordan-brough @pranavrajs @muhsin-k @sojan-official @nithindavid 

I have created the inbox with the twilio creds and also set up the sandbox. By pasting the callback URL. But then when I send hello or any other message , no conversation is getting created on chatwoot. Any idea where I am going wrong? 

Is there anything to do in .env file to twilio to work?

I tried postman using the callback URL as a POST method. It is showing me 204 error.  

Thanks in advance if you can help me in this.

**Comment by pranavrajs** [comment]
@Rishabhh17 A screencast and the server logs would be helpful to debug what exactly is going on. There is not much information here to debug it.
